packages/fxa-shared/l10n/branding.ftl

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### Localization for fxa
+
+## Brand and product terms to be used globally
+
+# Note: Terms should not be transliterated or translated
+# Declension should be avoided
+
+# Firefox and Mozilla Brand
+-brand-name-mozilla = Mozilla
+-brand-name-firefox = Firefox
+
+# Mozilla and Firefox products that should be treated as brands (not translated)
+-product-name-mozilla-vpn = Mozilla VPN
+-product-name-firefox-cloud = Firefox Cloud
+-product-name-firefox-monitor = Firefox Monitor
+-product-name-firefox-relay = Firefox Relay
+
+# Other brands
+-brand-name-apple = Apple
+-brand-name-google = Google
+-brand-name-paypal = PayPal
+-brand-name-pocket = Pocket
+-brand-name-stripe = Stripe
+
+# The following are not terms because they are not used directly in messages,
+# but rather looked up in code and passed into the message as variables.
+product-name-apple-app-store = App Store
+product-name-google-play = Google Play
+
+## FxA product name to be used globally
+
+# "Accounts" can be localized and should be lowercase, "Firefox" must be treated as a brand.
+# This is used to refer to the service as a whole OR to multiple accounts
+product-firefox-accounts = { -brand-firefox } accounts
```

Comment:
Q: Depending on results of previous conversations, should these be `-brand-name-firefox` (per L14 above)?
I think this and L42 are the only two references to `-brand-firefox` I see.

Reply:
Good point. 

In this file there should be terms, not normal messages. And no point in referencing the brand for Firefox.

See for example what locales are doing in Firefox
https://transvision.flod.org/string/?entity=browser/browser/branding/sync-brand.ftl:-fxaccount-brand-name&repo=gecko_strings